app/services/flash-messages.js

```diff
@@ -4,8 +4,9 @@ export default Service.extend({
   message: null,
   _nextMessage: null,
 
-  show(message) {
+  show(message, options = { type: 'warning' }) {
     this.set('message', message);
+    this.set('options', options);
   },
```

Comment:
are these changes still necessary? it looks like we're not using flash messages anymore

Reply:
If the project has no plan to bring better user exp with right info or success messages then I can remove it.